src/pages/ResearcherProfile.js

```diff
@@ -108,8 +110,9 @@ export const ResearcherProfile = hh(class ResearcherProfile extends Component {
   }
 
   async getResearcherProfile() {
+    const currentUser = Storage.getCurrentUser();
```

Comment:
Instead of calling ```Storage.getCurrentUser``` again, just pass in the ```currentUser``` you got on line 44 as a function argument.